src/main/java/duke/taskmanager/Deadline.java

```diff
@@ -0,0 +1,70 @@
+package duke.taskmanager;
+
+import duke.exceptions.emptyDescException;
+import duke.exceptions.unrecogException;
+import duke.exceptions.unspecTimeException;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Deadline class of parent class Tasks
+ */
+public class Deadline extends Tasks {
+    String icon = "[D]";
+
+    String desc;
+    String end;
+    LocalDate date;
+    String time;
```

Comment:
would it be better to add some kind of access modifiers to these variables? for example `protected`?